censusdis/data.py

```diff
@@ -1244,3 +1244,124 @@ def geographies(dataset: str, vintage: VintageType) -> List[List[str]]:
 
 
 variables = VariableCache()
+
+
+def _identify_counties(geo_df: gpd.GeoDataFrame, year: int):
```

Comment:
We normally used the naming convention `gdf_*` for geo data frames, so `gdf_geo` would be a better option here.